docs/CONTRIBUTING.md

```diff
@@ -8,7 +8,10 @@ Some suggestions to get started:
 - Help with packaging on various distributions needed!
 - To use print debugging to the [Helix log file][log-file], you must:
   * Print using `log::info!`, `warn!`, or `error!`. (`log::info!("helix!")`)
-  * Pass the appropriate verbosity level option for the desired log level. (`hx -v <file>` for info, more `v`s for higher severity inclusive)
+  * Pass the appropriate verbosity level option for the desired log level. (`hx -v <file>` for info, more `v`s for higher verbosity)
+  * Want to display the logs in a separate file instead of using the `:log-open` command in your compiled Helix editor? Start your debug version with `cargo run --log foo.log` and in a new terminal use `tail -f foo.log`
```

Comment:
```suggestion
  * Want to display the logs in a separate file instead of using the `:log-open` command in your compiled Helix editor? Start your debug version with `cargo run -- --log foo.log` and in a new terminal use `tail -f foo.log`
```

very minor point but without the double-dash, `cargo` thinks that the `--log` argument belongs to it instead of to `hx` so it prints usage and exits

Reply:
Ah, yes, indeed, i'll fix this straight away 👍